src/app.ts

```diff
@@ -0,0 +1,69 @@
+import * as bodyParser from "body-parser";
+import * as express from "express";
+import { Express } from "express";
+import { Pool } from "pg";
+
+import { log } from "./utils/logger";
+
+// Create Express server
+const app = express();
+
+const timeout = 5000;
+
+const dbParams = {
+  host: process.env.POSTGRES_HOST || "localhost",
```

Comment:
document these vars in the readme

Reply:
I suggest to have a single variable with the connection string, is more flexible

Reply:
@LucaPrete suggested to use three different environment variables, that's why I used them instead of a single one.

Reply:
unfortunately with the current code the database name is hardcoded (!)

besides, `pool` already gets the connection attributes from environment vars, see [the documentation](https://node-postgres.com/features/connecting#environment-variables)

Reply:
@cloudify @alexgpeppe I'm fine either way...as long as this is variable. I was slightly preferring to use three values instead of one, in order to give to the express container the same env variables given to the postgres container... at the end of the day the choice is up to you, as long as the value can be changed depending on some form of variables given in k8s or docker-file